corehq/apps/reports/views.py

```diff
@@ -1443,8 +1443,17 @@ def page_context(self):
 @require_form_view_permission
 @location_safe
 def view_form_attachment(request, domain, instance_id, attachment_id):
```

Comment:
where is this view used now? It looks like the 2 usages you changed where the only 2.

Reply:
Sorry, I should have mentioned. Its not used anywhere in HQ now but is only used by the project's app. They save the link to attachment as a case property. Should I just put this behind a feature flag to isolate its use case? Or just add documentation to this view to avoid getting it removed since it has no use within HQ anymore?

Reply:
Definitely adding a docstring is a good idea.

Is that project manually constructing the URL?

Reply:
Yes.
https://india.commcarehq.org/a/gcc-sangath/apps/view/c49be83c37954342b1ef73f46dca550a/form/64988fa8b253403cad5f635e8f843e4d/source/#form/recording_download_id